src/EAction.h

```diff
@@ -41,14 +41,15 @@ class EAction : public QAction
 public:
     EAction(QIcon&, QString&);
 
-    int mID;
-    QPointer<Host> mpHost;
-
 public slots:
     void slot_execute(bool checked);
 
 signals:
     void triggered(QAction*);
+
+public://private:
```

Comment:
Open the comment with TODO: that is one of the default term that the Qt Creator "Todo-list" plug-in tracks and indexes and can jump you with a click:

![qt_snapshot7](https://cloud.githubusercontent.com/assets/6163092/25210390/6bd7137e-2577-11e7-89d7-84c7e02676b1.png)